README.adoc

```diff
@@ -75,11 +75,11 @@ boolean isCiServer = System.getenv().containsKey("CI")
 test {
     if (isCiServer) {
         retry {
-            maxRetries = 3
+            maxRetries = 2
             maxFailures = 20
-            failOnPassedAfterRetry = true
         }
     }
+    failOnPassedAfterRetry = true
```

Comment:
This must still be within a `retry` block, doesn't it?

Reply:
Nice catch!